### PR TITLE
feat: page timeout

### PR DESCRIPTION
**Pull request by IonCaza**
### Description

Puts Navigation Timeout in a variable in config.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -12,6 +12,8 @@ const notifications = {
 	email: email.username && email.password
 };
 
+const navigationTimeout = 60000;
```
Comment (jef) [inline src/config.ts:15]: Let's put this inside the `page` variable

File: src/config.ts [hunk 2]
```diff
@@ -25,5 +27,6 @@ export const Config = {
 	notifications,
 	page,
 	rateLimitTimeout: 5000,
-	stores
+	stores,
+	navigationTimeout
```
Comment (jef) [inline src/config.ts:31]: Won't need this anymore if we put inside `page`.

```suggestion

```

**Review by jef — CHANGES_REQUESTED** [review]

File: src/config.ts [hunk 3]
```diff
@@ -15,7 +15,8 @@ const notifications = {
 const page = {
 	height: 1920,
 	userAgent: 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36',
-	width: 1080
+	width: 1080,
+	navigationTimeout: 60000
```
Comment (jef) [inline src/config.ts:19]: Oh! Let's also make this an environment variable and add to `README`.

File: src/config.ts (hunk 1 above)
Comment (IonCaza) [reply src/config.ts]: done

**Review by jef — CHANGES_REQUESTED** [review]

**Pull request merged**